Add MPTs to xrpl.org docs (#2908)
XRPLF/xrpl-dev-portal: docs/references/http-websocket-apis/public-api-methods/ledger-methods/ledger_entry.md

Return an `MPToken` object.

| Field                   | Type                       | Description           |
|:------------------------|:---------------------------|:----------------------|
| `mptoken`               | ️Object or String          | If a string, interpret as ledger entry ID of the MPToken to retrieve. If an object, requires the sub-fields account and mpt_issuance_id to unique identify the MPToken. |
| mptoken.mpt_issuance_id |	️String                      |	(Required if the `MPToken` is specified as an object) The 192-bit MPTokenIssuanceID that's associated with the MPTokenIssuance. |
| mptoken.account	️         | String	                    | (Required if the `MPToken` is specified as an object) The account that owns the MPToken. |

<!-- MULTICODE_BLOCK_START -->

*WebSocket*

```json
{
    "id": "example_get_mpt_issuance",
    "command": "ledger_entry",
    "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E",
    "ledger_index": "validated"
}
```

*JSON-RPC*

```json
{
  "method": "ledger_entry",
  "params": [{
    "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E",
    "ledger_index": "validated"
  }]
}
```
*Commandline*

```sh

> rippled json ledger_entry '{ "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E", "ledger_index": "validated" }'

Reviewer: these examples are for querying MPTokenIssuance object, not MPToken. Below is an example of querying an MPToken object
```

    "method": "ledger_entry",
    "params": [
        {
            "mptoken":{
                "mpt_issuance_id": "000002DFA4D893CFBC4DC6AE877EB585F90A3B47528B958D",
                "account":"r33kves44ksufkHSGg3M6GPPAsoVHEN8C1"
            } 
        }
    ]
```

| mptoken.mpt_issuance_id |	️String                      |	(Required if the `MPToken` is specified as an object) The 192-bit MPTokenIssuanceID that's associated with the MPTokenIssuance. |
| mptoken.account	️         | String	                    | (Required if the `MPToken` is specified as an object) The account that owns the MPToken. |

<!-- MULTICODE_BLOCK_START -->

*WebSocket*

```json
{
    "id": "example_get_mpt_issuance",
    "command": "ledger_entry",
    "mpt_issuance_id": "000002DFA4D893CFBC4DC6AE877EB585F90A3B47528B958D",
    "account":"r33kves44ksufkHSGg3M6GPPAsoVHEN8C1"
}
```

*JSON-RPC*

```json
{
    "method": "ledger_entry",
    "params": [
        {
            "mptoken":{
                "mpt_issuance_id": "000002DFA4D893CFBC4DC6AE877EB585F90A3B47528B958D",
                "account":"r33kves44ksufkHSGg3M6GPPAsoVHEN8C1"
            } 
        }
    ]
}
```
*Commandline*

```sh
rippled json ledger_entry '{ "mpt_issuance_id": "000002DFA4D893CFBC4DC6AE877EB585F90A3B47528B958D", "account":"r33kves44ksufkHSGg3M6GPPAsoVHEN8C1"}'
```
<!-- MULTICODE_BLOCK_END -->

> [Try it! >](https://xrpl.org/resources/dev-tools/websocket-api-tool.html#ledger_entry-mpt_issuance)

Reviewer 2: ```suggestion
[Try it! >](/resources/dev-tools/websocket-api-tool.html#ledger_entry-mpt_issuance)
```

> [Try it! >](https://xrpl.org/resources/dev-tools/websocket-api-tool.html#ledger_entry-mpt_issuance)

Reviewer 2: ```suggestion
[Try it! >](/resources/dev-tools/websocket-api-tool#ledger_entry-mpt_issuance)
```

## Response Format

> The response follows the [standard format](https://xrpl.org/docs/references/http-websocket-apis/api-conventions/response-formatting/), with a successful result containing the following fields:

Reviewer 2: ```suggestion
The response follows the [standard format][], with a successful result containing the following fields:
```

| Field          | Type             | Description                              |
|:---------------|:-----------------|:-----------------------------------------|
| `index`        | String           | The unique ID of this [ledger entry](https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/). |
| `ledger_index` | Unsigned Integer | The [ledger index](https://xrpl.org/docs/references/protocol/data-types/basic-data-types/#ledger-index) of the ledger that was used when retrieving this data. |
| `node`         | Object           | _(Omitted if `"binary": true` specified.)_ Object containing the data of this ledger entry, according to the [ledger format](https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/). |

> | `node_binary`  | String           | _(Omitted unless `"binary":true` specified)_ The [binary representation](https://xrpl.org/docs/references/protocol/binary-format/) of the ledger object, as hexadecimal. |

Reviewer 2: ```suggestion
| `index`        | String           | The unique ID of this [ledger entry](../../../protocol/ledger-data/ledger-entry-types/index.md). |
| `ledger_index` | Unsigned Integer | The [ledger index](../../../protocol/data-types/basic-data-types/index.md#ledger-index) of the ledger that was used when retrieving this data. |
| `node`         | Object           | _(Omitted if `"binary": true` specified.)_ Object containing the data of this ledger entry, according to the [ledger format](../../../protocol/ledger-data/ledger-entry-types/index.md). |
| `node_binary`  | String           | _(Omitted unless `"binary":true` specified)_ The [binary representation](../../../protocol/binary-format/index.md) of the ledger object, as hexadecimal. |
```

> | `node_binary`  | String           | _(Omitted unless `"binary":true` specified)_ The [binary representation](https://xrpl.org/docs/references/protocol/binary-format/) of the ledger object, as hexadecimal. |

Reviewer 2: ```suggestion
| `index`        | String           | The unique ID of this [ledger entry](../../../protocol/ledger-data/ledger-entry-types/index.md). |
| `ledger_index` | Unsigned Integer | The [ledger index](../../../protocol/data-types/basic-data-types.md#ledger-index) of the ledger that was used when retrieving this data. |
| `node`         | Object           | _(Omitted if `"binary": true` specified.)_ Object containing the data of this ledger entry, according to the [ledger format](../../../protocol/ledger-data/ledger-entry-types/index.md). |
| `node_binary`  | String           | _(Omitted unless `"binary":true` specified)_ The [binary representation](../../../protocol/binary-format.md) of the ledger object, as hexadecimal. |
```
